### DISCUSSION TITLE
Way to deal MP damage? Or have abilities like encourage have a MP cost tied into using them?

### DISCUSSION THREAD

**Discussion by Pattern-Weld** [League-of-Fabulous-Developers/FoundryVTT-Fabula-Ultima#112] [Q&A] [answered]
Just wanted to make sure I wasn't missing something, is there anything yet that automates MP cost or allows an attack to deal MP damage like Desdemona in the Quick Start?

**Comment by spyrella** [comment, marked as answer]
Sorry for late reply, there is currently no automated way to deduct mp cost. The current approach to deducting mp is adding `@LOSS[X mp]` into the description and drag and dropping it the token. 
![image](https://github.com/League-of-Fabulous-Developers/FoundryVTT-Fabula-Ultima/assets/16845165/631d962d-817d-4852-b8aa-a28328f50301)

**Reply by Pattern-Weld** [reply]
Thank you so much! This example will be a great help as I fill in future abilities
